opennms-doc/guide-admin/src/asciidoc/text/events/eif-adapter.adoc

```diff
@@ -0,0 +1,53 @@
+
+// Allow GitHub image rendering
+:imagesdir: ../../images
+
+[[ga-events-eif]]
+=== Receiving IBM Tivoli Event Integration Facility Events
+
+_OpenNMS_ can be configured to receive _Events_ sent using the https://www.ibm.com/support/knowledgecenter/SSSHTQ_7.3.1/com.ibm.netcool_OMNIbus.doc_7.3.1/omnibus/wip/eifsdk/concept/kaa24487.html[Tivoli Event Integration Facility].
+These EIF events are translated into OpenNMS events using preconfigured rules. The resulting UEI are anchored in the `uei.opennms.org/vendor/IBM/EIF/` namespace, with the name of the EIF https://www.ibm.com/support/knowledgecenter/SSSHTQ_7.3.1/com.ibm.netcool_OMNIbus.doc_7.3.1/omnibus/wip/eifsdk/concept/ecoemst16.html[event class] appended.
+
+A sample event configuration for the OMEGAMON_BASE class is included with OpenNMS.
+
+[[ga-events-eif-configuring]]
+==== Configuring
+First check that your _OpenNMS_ installation supports this feature.
```

Comment:
We'll need to update this section now that the config file is gone.

You can assume that if this content is in the documentation for the corresponding version, then the feature will be present.